static/translations/messages_en.json

```diff
@@ -47,7 +47,10 @@
   "Narrow to conversation": "Narrow to conversation",
   "Reply": "Reply",
   "Add a reaction": "Add a reaction",
-  "Copy to clipboard": "Copy to clipboard",
+  "Copy message content": "Copy message content",
+  "Copied message content": "Copied message content",
+  "Copy message link": "Copy message link",
+  "Copied link to message": "Copied link to message",
```

Comment:
I think there's a few more new/edited strings, like "Failed to copy message content to clipboard".

Reply:
Good catch, thanks